test/common/inspector-helper.js

```diff
@@ -168,9 +168,7 @@ class InspectorSession {
         reject(message.error);
     } else {
       if (message.method === 'Debugger.scriptParsed') {
-        const script = message['params'];
-        const scriptId = script['scriptId'];
-        const url = script['url'];
+        const { scriptId, url } = message.params;
```

Comment:
This doesn't seem correct.

Reply:
I think it is, no? `message['params']['scriptId']` and `message['params']['url']`.